sideline-models/kafka_http_sideline_meta.go

```diff
@@ -0,0 +1,6 @@
+package sideline_models
+
+type KafkaHttpSidelineMeta struct {
```

Comment:
Golang doesn't require to define structs/interface in separate files. They can all reside in single file, along with the usage